### ISSUE TITLE
Observatory has missing alt text on an image

### ISSUE THREAD

**Issue by globau** [mdn/fred#521]
Accessibility report: observatory has missing alt text on an image:

> When I scrolled past the scanner buttons, I noticed an image with no alt text. I don't know what this image is referring to because of a lack of information.

Perhaps this is the `landing-illustration`?

**Comment by argl** [comment]
Illustrative SVGs lack any usefull a11y-related attributes, recommended approach is to add `role="presentation"` to those images.
